build.js

```diff
@@ -23,7 +23,8 @@ import * as rollup from 'rollup';
   if (result_es5.emitSkipped) {
     const diagnostic = result_es5.diagnostics[0];
     const error = new Error(diagnostic.messageText);
-    error.stack = error.stack.replace(/at .*/, 'at ' + diagnostic.file.fileName + ':' + diagnostic.file.lineMap.findIndex(value => value > diagnostic.start));
+    //error.stack = error.stack.replace(/at .*/, 'at ' + diagnostic.file.fileName + ':' + diagnostic.file.lineMap.findIndex(value => value > diagnostic.start));
+    console.error(diagnostic);
```

Comment:
It's not super important but could you either update the stack in a way that works (if it's the line map that's problematic, maybe just remove that part?) or remove that line altogether, instead of just commenting it out?